x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/chart_switch/chart_switch.tsx

```diff
@@ -405,6 +405,7 @@ export const ChartSwitch = memo(function ChartSwitch({
           isPreFiltered
           data-test-subj="lnsChartSwitchList"
           searchProps={{
+            autoFocus: true,
```

Comment:
I'll remove it from here and move to my PR here: https://github.com/elastic/kibana/pull/187475/commits/01ad33d9860730fa7c6fd3ba1b8e52e5831feee9
to avoid conflicts later on 🙏🏼 
```suggestion
```